demo/molecule/modal/demo/MobileFitContentModal.js

```diff
@@ -0,0 +1,52 @@
+/* eslint react/prop-types: 0 */
+import React from 'react'
+import MoleculeModal from '../../../../components/molecule/modal/src'
+import {
+  ContentWithCloseButton,
+  LoremIpsumParagraph,
+  IconClose
+} from './helperComponents'
+
+export default class MobileFitContentModal extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      open: false
+    }
+  }
+
+  handleOpenModal = () => {
+    this.setState({
```

Comment:
again, make it be in 1 line manually and save, it should stay like that because it's less than 80 charrs

Reply:
It does not apply to this PR